todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/view/AddEditTaskViews.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016, The Android Open Source Project
+ * Copyright (c) 2017-2018 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.architecture.blueprints.todoapp.addedittask.view
+
+import android.support.design.widget.FloatingActionButton
+import android.support.design.widget.Snackbar
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import com.example.android.architecture.blueprints.todoapp.R
+import com.example.android.architecture.blueprints.todoapp.addedittask.domain.Event
+import com.example.android.architecture.blueprints.todoapp.addedittask.domain.Model
+import com.example.android.architecture.blueprints.todoapp.addedittask.domain.TaskDefinitionCompleted
+import com.example.android.architecture.blueprints.todoapp.util.onNew
+import com.spotify.mobius.Connectable
+import com.spotify.mobius.Connection
+import com.spotify.mobius.functions.Consumer
+
+class AddEditTaskViews(inflater: LayoutInflater, parent: ViewGroup, private val mFab: FloatingActionButton)
+    : Connectable<Model, Event> {
+    val rootView: View = inflater.inflate(R.layout.addtask_frag, parent, false)
+    private val mTitle: TextView
+    private val mDescription: TextView
+
+    init {
+        mTitle = rootView.findViewById(R.id.add_task_title)
+        mDescription = rootView.findViewById(R.id.add_task_description)
```

Comment:
Consider Kotlin Android Extensions viewbinding?

Reply:
hmm would it work for a non-Android class though? The examples show an `Activity`

Reply:
On https://kotlinlang.org/docs/tutorials/android-plugin.html , look under "LayoutContainer"